### ISSUE TITLE
Can't connect to RPC server

### ISSUE THREAD

**Issue by ZeQing-Liu** [laike9m/Cyberbrain#96]
when i use cyberbrain ，terminal shows can't connect to RPC server。 mac os 11

**Comment by laike9m** [comment]
Please follow the bug report template and provide necessary information.
https://github.com/laike9m/Cyberbrain/issues/new?assignees=&labels=bug&template=bug_report.md&title=

**Comment by leon0204** [comment]
same problem with u , i think u cyberbrain version in pip may different  from  u cyberbrain version in vscode . 
@ZeQing-Liu 

I change version match slove this problem

**Comment by fvbrandt** [comment]
I have the same issue and I have the same version for both the python package and the vsc extension.
The code runs through and at the very end I get "Can't connect to RPC server".
What could be the reason? What can I do to help debug this?

Versions:
- OS you're using: Windows 10 Enterprise
- Python version: Python 3.7.4
- Cyberbrain Python lib version: 0.1.4
- Cyberbrain VS Code extension version: 0.1.4

**Comment by laike9m** [comment]
Given there's enough information, I can take a look this weekend. @ZeQing-Liu, @leon0204, @fvbrandt, please provide a code snippet that can reproduce this error so I can debug.

**Comment by laike9m** [comment]
I'll close this issue next week since I can't debug it without a way to reproduce.

**Comment by fvbrandt** [comment]
Hi @laike9m, sorry for not replying earlier. I have the issue on my work PC, hence I didn't reply over the weekend.
The good news is that I found the problem by looking at developer tools as you suggested in the other issue:
The extension could not be activated because it tried to create c:\tmp. On my work PC I need elevated permissions to create a directory in c:\. Once I manually created c:\tmp and gave full permissions to that directory to my non-elevated user, it worked.
I suggest to use https://docs.python.org/3/library/tempfile.html instead of c:\tmp?
